src/webgpu/api/operation/resource_init/buffer.spec.ts

```diff
@@ -1,11 +1,231 @@
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { assert, unreachable } from '../../../../common/util/util.js';
+import { GPUConst } from '../../../constants.js';
 import { GPUTest } from '../../../gpu_test.js';
 
 export const description = `
 Test uninitialized buffers are initialized to zero when read
 (or read-written, e.g. with depth write or atomics).
 
-TODO
+Here lists all the buffer usages to test in future patches:
```

Comment:
nit: Technically I think this list still needs a TODO so that we know these items are to be done.